.github/actions/run-android-device-farm-test/action.yml

```diff
@@ -22,6 +22,7 @@ runs:
       with:
         project_arn: ${{ inputs.project-arn}}
         device_pool_arn: ${{ inputs.device-pool-arn }}
+        timeout: 2700
```

Comment:
Will that be enough - the others got bumped from 45 to 50.

Reply:
The tests themselves usually take a little more than 40 min, that usually was going after the limit of 45 minutes including the time to build and some other work. This timeout here is only for the tests, so I thought that 45 minutes (2700 seconds) would be enough. I just increased it a little bit more to be on the safe side.